includes/class-post-type-offer.php

```diff
@@ -0,0 +1,1289 @@
+        add_action( 'init', array( $this, 'register_taxonomy' ) );
+        $this->add_meta_box();
+    }
+
+    /**
+     * Register post type
+     *
+     * @return void
+     */
+    public function register_post_type() {
+        $show_in_menu = true;
+        register_post_type( 'offer', array(
+            'label'           => __( 'Offers', 'bediq' ),
+            'public'          => true,
+            'show_ui'         => true,
+            'show_in_menu'    => $show_in_menu,
+            'menu_position'   => 6,
+            'capability_type' => 'post',
+            'hierarchical'    => false,
+            'rewrite'         => array('slug' => 'offers'),
+            'query_var'       => true,
+            'has_archive'     => true,
+            'supports'        => array('title', 'editor', 'thumbnail'),
+            'menu_icon'       => 'dashicons-tickets-alt',
+            'taxonomies'      => [ 'hotel_offers' ],
+            'labels'          => array(
+                'name'               => __( 'Offers', 'bediq' ),
+                'singular_name'      => __( 'Offer', 'bediq' ),
+                'menu_name'          => __( 'Offers', 'bediq' ),
+                'add_new'            => __( 'Add Offer', 'bediq' ),
+                'add_new_item'       => __( 'Add New Offer', 'bediq' ),
+                'edit'               => __( 'Edit', 'bediq' ),
+                'edit_item'          => __( 'Edit Offer', 'bediq' ),
+                'new_item'           => __( 'New Offer', 'bediq' ),
+                'view'               => __( 'View Offer', 'bediq' ),
+                'view_item'          => __( 'View Offer', 'bediq' ),
+                'search_items'       => __( 'Search Offers', 'bediq' ),
+                'not_found'          => __( 'No Offers Found', 'bediq' ),
+                'not_found_in_trash' => __( 'No Offers Found in Trash', 'bediq' ),
+                'parent'             => __( 'Parent Offer', 'bediq' ),
+            ),
+            'rewrite'          =>  array(
+                'slug'               => 'offers',
+                'with_front'         => true,
+                'pages'              => true,
+                'feeds'              => true,
+            ),
+        ) );
+    }
+
+    /**
+     * Register taxonomy
+     *
+     * @return void
+     */
+    public function register_taxonomy() {
+        register_taxonomy( 'offer_types', array( 'offer' ),
+            array(
+                'hierarchical'   => true,
+                'label'          => __( 'Offer Types', 'bediq' ),
+                'show_ui'        => true,
+                'query_var'      => true,
+                'rewrite'        => array('slug' => 'hotel-offers'),
+                'singular_label' => __( 'Offers', 'bediq' )
+            )
+        );
+    }
+
+    /**
+     * Add meta box
+     */
+    public function add_meta_box() {
+        if ( function_exists('acf_add_local_field_group') ) {
```

Comment:
Instead of indenting the whole function, it'll be more easier like this:

```
if ( !function_exists('acf_add_local_field_group') ) {
    return;
}
```